packages/api/test/e2e/promise-queries.spec.js

```diff
@@ -168,4 +168,19 @@ describe.skip('e2e queries', () => {
       done();
     });
   });
+
+  it('queries state using double map key', async () => {
+    // TODO Update ['any', '0x1234'] to the key of a known event topic and update '[]' to the expected value
+    const eventTopics = await api.query.system.eventTopics(['any', '0x1234']);
+
+    expect(eventTopics.toString()).toEqual('[]');
+  });
+
+  it('subscribes to queries using double map key', async (done) => {
+    // TODO Update ['any', '0x1234'] to the key of a known event topic and update '[]' to the expected value
+    api.query.system.eventTopics(['any', '0x1234'], (eventTopics) => {
+      expect(eventTopics.toString()).toEqual('[]');
+      done();
+    });
+  });
```

Comment:
I added an e2e test here, now it actually doesn't test anything here... But if we update ['any', '0x1234'] to the key of a known event topic, it will test the double map.

Reply:
Yes, in but not fully used. Cannot win them all. It is fine, not perfect, but fine. (Since you have verified against a real supported chain, will accept that - and the above gives the gap to properly extend once substrate moves along a bit)

Reply:
Thanks. Yeah, we could extend this later.